pages/docs/hooks.md

```diff
@@ -33,7 +33,7 @@ Code Example:
 
 ```go
 func (u *User) BeforeSave() (err error) {
-	if u.IsValid() {
+	if !u.IsValid() {
```

Comment:
not a huge deal, but I lost a bit of time when copying this logic until I realised we had it wrong here